src/Raven.Server/Documents/Indexes/Sharding/Persistence/Corax/ShardedCoraxIndexReadOperation.cs

```diff
@@ -131,6 +145,22 @@ private ShardedQueryResultDocument AddOrderByFields(Document queryResult, IndexQ
 
         return result;
     }
+
+    private static bool CompareCoraxAndQueryOrderFieldType(MatchCompareFieldType corax, OrderByFieldType query)
```

Comment:
`Compare....()` methods usually return some value e.g. an integer. Since you return `bool` then maybe `IsSameOrderType(MatchCompareFieldType matchCompareFieldType, OrderByFieldType orderByFieldType)` could be more suitable here

Reply:
Changed